projects/components/src/number-input/number-input.component.scss

```diff
@@ -23,6 +23,14 @@
   border-bottom: 1px solid $gray-2;
 }
 
+.no-border {
```

Comment:
It should be no border by default (i think)

Reply:
Yep, I think the same but there was used previously in many places so for this reason, a created a new style

Reply:
i mean, if we don't pass any style then would it not hide the borders on its own?

Reply:
There are two previous scenarios 
1 - Border: sets all borders
2- Underline: sets just the border-bottom

It's necessary to add a new Appearance to have no border input because Border is set by default currently. We could remove the default border but could affect multiple places through the app.